smile_web_auto_refresh/static/src/js/web_auto_refresh.js

```diff
@@ -62,7 +62,7 @@ odoo.define('web_auto_refresh', function (require) {
                     } else if (typeof(widget.controllers) != 'undefined') {
                         var controller = widget.getCurrentController();
                         var action = widget.getCurrentAction();
-                        if (action && action.auto_search && controller.widget.modelName == message && controller.widget.mode != "edit") {
+                        if (action && action.auto_search && controller.widget.modelName == message && controller.widget.mode != "edit" && controller.viewType !== 'form') {
                             controller.widget.reload();
```

Comment:
Make sure, `logistics_pricing_import_thread` is not affected by this change. Why not extended on your custom module?

Reply:
Why would it affect `logistics_pricing_import_thread`? You call super, so it should work fine. Otherwise there is something else I don't know? 

"Why not extended on your custom module?". This is general change, so no need to hack on custom module. And it would be clunky to do on custom module, because I would not be able to call `super` as it would trigger original refresh, which I need to limit.